site/source/docs/getting_started/FAQ.rst

```diff
@@ -404,8 +404,8 @@ what you want on the web: even though ``main()`` exited, you may have something
 asynchronous happening later that you want to execute.
 
 In some cases, though, you may want a more "commandline" experience, where we do
-shut down the runtime when ``main()`` exits. You can build with ``-s
-EXIT_RUNTIME``, and then we will call ``atexits`` and so forth. When you build
```

Comment:
Ah good catch.. because it was split over multiple lines my grep didn't catch this one.